index.cjs

```diff
@@ -782,11 +782,13 @@ safeql: try {
       ],
     },
   ];
-} catch (err) {
-  // Swallow errors in non-CI environments to avoid noisy ESLint logs in non-database projects
-  if (process.env.CI) {
-    throw err;
-  }
+} catch (error) {
+  throw Object.assign(
```

Comment:
why the `Object.assign`?

Reply:
ok yeah, that's not what we are looking for, see if you can find a different way of doing this (we have wrapped other errors a few times across our codebases)